### PR TITLE
Adds ancient toolbox

### PR DESCRIPTION
**Pull request by CoolCatIsFeelingGroovy**
## About The Pull Request

Adds the ancient toolbox from TG.

An assistant only traitor item costing 2 TC, comes with a basic set of tools and a pair of insulated gloves.
Starts with a melee force of 19, but increases by 1 for every TC inside.

## Why It's Good For The Game

Assistants don't get any interesting role-restricted items. It also makes traitor assistant more fair as helps make up for their lack of access and items that they get from a job.

## Changelog
:cl:
add: Added the ancient toolbox, an assistant only traitor item that does increased damage based on the amount of telecrystals it contains.
/:cl:

### CODE REVIEW

File: code/game/objects/items/storage/toolbox.dm [hunk 1]
```diff
@@ -94,6 +94,40 @@ GLOBAL_LIST_EMPTY(rubber_toolbox_icons)
 /obj/item/storage/toolbox/mechanical/old/heirloom/PopulateContents()
 	return
 
+
+/obj/item/storage/toolbox/mechanical/old/clean
+	name = "toolbox"
+	desc = "A old, blue toolbox, it looks robust."
```
Comment (CornMyCob) [inline code/game/objects/items/storage/toolbox.dm]: An old

Comment (AffectedArc07) [reply code/game/objects/items/storage/toolbox.dm]: Please either mark these as Kepler changes or put them into the modular folder

**Comment by AffectedArc07** [comment]
The sprite for this **needs** to be in its own icon file in the modular folder because icon conflicts are extremely annoying to resolve.

**Comment by AffectedArc07** [comment]
```
code/modules/unit_tests/component_tests.dm, line 12, column 3:
352error: undefined proc: "Fail" on /datum/unit_test/component_duping
```
What did you even do?

File: tgstation.dme [hunk 2]
```diff
@@ -2840,6 +2840,7 @@
 #include "code\modules\tgui\states\zlevel.dm"
 #include "code\modules\tooltip\tooltip.dm"
 #include "code\modules\unit_tests\_unit_tests.dm"
+#include "code\modules\unit_tests\component_tests.dm"
```
Comment (AffectedArc07) [inline tgstation.dme]: I think this got accidentally ticked

Comment (CoolCatIsFeelingGroovy) [reply tgstation.dme]: FUCK

**Pull request merged**